test/remote/check_tests.js

```diff
@@ -28,9 +28,15 @@ test(
 
 var client = restify.createJsonClient({
   url: 'http://127.0.0.1:' + config.listen.port
-});
+})
+
+var ACTIONS = [
+  'accountCreate',
+  'accountLogin',
+  'passwordChange'
+]
 
-['accountCreate', 'accountLogin', 'passwordChange'].forEach(function (action) {
+ACTIONS.forEach(function (action) {
```

Comment:
@pdehaan Should we still be leaving out semi-colons ;) 
@kitcambridge : That was awesome, my dear sir!